### ISSUE TITLE
[Bug]: source: sysav_se stopped working?

### ISSUE THREAD

**Issue by maglerod** [mampfes/hacs_waste_collection_schedule#1536]
### I Have A Problem With:

A specific source

### What's Your Problem

Since last versions of HA i'm getting error i the log and the result is only presented as "unknown / okand"

It has worked before, and i've also verified that sysav_se still works and that my address also works.

### Source (if relevant)

sysav_se

### Logs

```Shell
Logger: waste_collection_schedule.source_shell
Source: custom_components/waste_collection_schedule/waste_collection_schedule/source_shell.py:136
Integration: waste_collection_schedule ([documentation](https://github.com/mampfes/hacs_waste_collection_schedule#readme))
First occurred: 07:36:56 (1 occurrences)
Last logged: 07:36:56

fetch failed for source Sysav Sophämntning: Traceback (most recent call last): File "/config/custom_components/waste_collection_schedule/waste_collection_schedule/source_shell.py", line 134, in fetch entries = self._source.fetch() ^^^^^^^^^^^^^^^^^^^^ File "/config/custom_components/waste_collection_schedule/waste_collection_schedule/source/sysav_se.py", line 29, in fetch address_data = json.loads(response.text) ^^^^^^^^^^^^^^^^^^^^^^^^^ File "/usr/local/lib/python3.11/json/__init__.py", line 346, in loads return _default_decoder.decode(s) ^^^^^^^^^^^^^^^^^^^^^^^^^^ File "/usr/local/lib/python3.11/json/decoder.py", line 337, in decode obj, end = self.raw_decode(s, idx=_w(s, 0).end()) ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ File "/usr/local/lib/python3.11/json/decoder.py", line 355, in raw_decode raise JSONDecodeError("Expecting value", s, err.value) from None json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

### Relevant Configuration

```YAML
in configuration.yaml

waste_collection_schedule:
  sources:
    - name: sysav_se
      args:
        street_address: (my address...)






  - platform: waste_collection_schedule
    name: Soptomning
```

### Checklist Source Error

- [ ] Use the example parameters for your source (often available in the documentation) (don't forget to restart Home Assistant after changing the configuration)
- [X] Checked that the website of your service provider is still working
- [X] Tested my attributes on the service provider website (if possible)
- [X] I have tested with the latest version of the integration (master) (for HACS in the 3 dot menu of the integration click on "Redownload" and choose master as version)

### Checklist Sensor Error

- [ ] Checked in the Home Assistant Calendar tab if the event names match the types names (if types argument is used)

### Required

- [X] I have searched past (closed AND opened) issues to see if this bug has already been reported, and it hasn't been.
- [X] I understand that people give their precious time for free, and thus I've done my very best to make this problem as easy as possible to investigate.

**Comment by 5ila5** [comment]
fixed by #1538 
Fix will be part of release 1.45.0 or is available now as master (git) version (HACS: 3-dot menu -> redownload -> change version to master)

**Comment by maglerod** [comment]
Wow! Thankyou for quick reply and i downloaded the "master" and it worked. 
Thanks!
